README.md

```diff
@@ -1,37 +1,77 @@
-# Welcome
+Cloudinary URL-GEN SDK
+=========================
+[![Build Status](https://api.travis-ci.com/cloudinary/js-url-gen.svg?branch=master)](https://app.travis-ci.com/github/cloudinary/js-url-gen)
+## About
+The Cloudinary URL-GEN SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
 
-## About this project
+This SDK can also be used with [popular frontend frameworks](https://cloudinary.com/documentation/sdks/js/frontend-frameworks/index.html).
 
-This project enables you to create Cloudinary URLs for your images and videos.
-Using this SDK, you can apply advanced transformations to your images and videos.
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [URL-Gen SDK Guide](https://cloudinary.com/documentation/javascript_integration) and the [URL-GEN Reference](https://cloudinary.com/documentation/sdks/js/url-gen/index.html).
+
+
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generate Image and Video URLs](#Generate-Image-and-Video-URLs)
+    - [Transpilation](#Transpilation)
+    - [Testing with Jest](#Testing-with-jest)
+- [Contributions](#Contributions)
+- [Get Help](#Get-Help)
+- [Additional Resources](#Additional-Resources)
+
+## Key Features
+- [Transform image](https://cloudinary.com/documentation/javascript_image_transformations) assets (links to docs).
+- [Transform video](https://cloudinary.com/documentation/javascript_video_transformations) assets (links to docs).
+
+
+## Version Support
+
+### Note!
+This SDK is cross-platform, but only the node versions are worth mentioning
+
+| SDK Version   | Node 10    | Node 12  | Node 12  |Node 14   | Node 16  |
+|---------------|------------|----------|----------|----------|----------|
+| 1.0.0 - 1.5.1 | V          | V        | V        | V        | V        |
 
-This SDK can also be used with [popular frontend frameworks](https://cloudinary.com/documentation/sdks/js/frontend-frameworks/index.html).
 
-## Help & Examples
-- [Getting started](https://cloudinary.com/documentation/sdks/js/url-gen/tutorial-gettingStarted.html)
-- [Annotated Code Examples](https://cloudinary.com/documentation/sdks/js/url-gen/tutorial-annotatedExamples.html)
-- [Configuration](https://cloudinary.com/documentation/sdks/js/url-gen/tutorial-configuration_.html)
 
 ## Installation
+### Install using your favorite package manager (yarn, npm)
 ```bash
-npm install @cloudinary/url-gen 
+npm install @cloudinary/url-gen
```

Comment:
Please split into 2 separate snippets so that the copy to clipboard button is usable